uber/templates/accounts/change_password.html

```diff
@@ -34,9 +34,8 @@ <h2 class="center"> Change Password </h2>
     $(function() {
     $(document.body).on('click', '.submit-button', function (event) {
         event.preventDefault();
-        if ($('#new1').val() == $('#new2').val()){
             $(event.target).parents('form').submit();
-        }
+        
```

Comment:
@Utilityman: Can you delete the entire contents of this ``<script>...</script>`` block?  The entire thing is unnecessary now that we're doing the password-match check on the server side, and the code as written now will result in a Javascript syntax error.  (The error won't stop the page from working, but it might confuse someone who opened the Javascript console or tried to debug something else on the page.)